File: PYTHON/Daily Flash/Week2/Day1_Jan21/code/prog1.py
'''
Program 1: Write a program to print squares of first 10 numbers.
Output:
Square of 1 : 1
Square of 2: 4
Square of 3 : 9
.
.
.
Square of 1 0 : 1 00
'''

for x in range(10):
    print("Square of "+ str(x+1) + ": " + str ((x+1)* (x+1)))

File: C++/Daily Flash/week1/Day6/code_C++/prog1.cpp
/*
 * Program 1: Write a program to print First 10 Natural Numbers.
Output: 1 2 3 4 5 6 8 9 10
*
*/

#include <iostream>

int main() {

	std::cout << "First 10 Natural num:" << std::endl;
	for(int i=1 ;i <= 10 ;i++)
		std::cout << i << " ";
	std::cout << std::endl;
	return 0;
}
